cmd/kubectltranslate/kubectltranslate.go

```diff
@@ -0,0 +1,134 @@
+/*
+Copyright IBM Corporation 2020
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"os"
+	"path/filepath"
+
+	cmdcommon "github.com/konveyor/move2kube/cmd/common"
+	"github.com/konveyor/move2kube/internal/common"
+	"github.com/konveyor/move2kube/internal/move2kube"
+	"github.com/konveyor/move2kube/internal/qaengine"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+type translateFlags struct {
+	outpath  string
+	srcpath  string
+	name     string
+	qacaches []string
+}
+
+var verbose bool
+
+func translateHandler(cmd *cobra.Command, flags translateFlags) {
+	// Setup
+	var err error
+	ignoreEnv := false
```

Comment:
Let's add a comment why we are setting ignoreenv as false

Reply:
done